### PR TITLE
DOCS: How to remove users from the hub

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://2i2c.freshdesk.com/a/tickets/61

### CODE REVIEW

File: admin/howto/manage-users.md [hunk 1]
```diff
@@ -79,6 +79,17 @@ Fill in usernames and optionally make them administrators. You can add multiple
 `````
 
 
+### To remove users
```
Comment (choldgraf) [inline admin/howto/manage-users.md:82]:
```suggestion
### Remove users from a hub
```

File: admin/howto/manage-users.md [hunk 2]
```diff
@@ -79,6 +79,17 @@ Fill in usernames and optionally make them administrators. You can add multiple
...
+
+You can revoke a user's access to the JupyterHub by removing them from the admin panel.
```
Comment (choldgraf) [inline admin/howto/manage-users.md:84]:
```suggestion
You can revoke a user's access to the JupyterHub by removing them from the allowed user list, using the admin panel.
```

File: admin/howto/manage-users.md [hunk 3]
```diff
@@ -79,6 +79,17 @@ Fill in usernames and optionally make them administrators. You can add multiple
...
+
+1. Find the name of the user in your admin panel. If they have a running server,
+   click the {guilabel}`Stop server` button to stop their running server.
+2. Click the {guilabel}`Delete user` button for this user. Confirm the deletion in
+   the dialog box that pops up.
+
+After this, the user would not be able to log in.
```
Comment (choldgraf) [inline admin/howto/manage-users.md:91]: will this delete their files as well? We should clarify this as I suspect admins will wonder

**Comment by choldgraf** [comment]
a few minor questions from me, looks good though!

File: admin/howto/manage-users.md (hunk 3 above)
Comment (yuvipanda) [reply admin/howto/manage-users.md:91]:
```suggestion
After this, the user would not be able to log in. However, their files will not be deleted -
if you add them later, their files will still be present.
```

**Comment by yuvipanda** [comment]
@choldgraf ty for the suggestions! Done!

**Comment by ericvd-ucb** [comment]
Hi - I am not sure where to leave a comment - as I cannot see freshdesk and some of this is in an email. 

I think the admin of this hub would like to lock up the hub, stop incoming login requests from users, but still be able to access user files as admin.   And not delete users one by one I guess.

**Comment by yuvipanda** [comment]
Ah, thanks @ericvd-ucb. That freshdesk ticket is the one you are on the email thread for. I've updated that with next steps.

**Pull request merged**